workflows/colabfold.nf

```diff
@@ -91,7 +91,7 @@ workflow COLABFOLD {
         //
         // MODULE: Run mmseqs
         //
-        if (params.colabfold_model_preset != 'AlphaFold2-ptm') {
+        if (params.colabfold_model_preset != 'alphaFold2-ptm') {
```

Comment:
It should be alphafold2_ptm, all lowercase, with underscore for the latest version I think.